backend/src/movements.rs

```diff
@@ -157,6 +255,13 @@ fn classify_relative_bearing(abs_bearing1: f64, abs_bearing2: f64) -> &'static s
     }
 }
 
+// Given two angles in [0, 360), return the smallest rotation between them
```

Comment:
I am so bad at reasoning about angles, but I think this is simple and correct, and the unit tests agree...